base/deprecated.jl

```diff
@@ -254,3 +254,18 @@ end
 @deprecate catch_stack(task=current_task(); include_bt=true) current_exceptions(task; backtrace=include_bt) false
 
 # END 1.7 deprecations
+
+# BEGIN 1.8 deprecations
+
+macro _inline_meta()
+    f, l = __source__.file, __source__.line
+    @warn "`Base.@_inline_meta` at $f:$l is deprecated, use the no-arg `@inline` annotation instead."
```

Comment:
I think this should use `depwarn(...)` instead.

Reply:
```suggestion
    depwarn("`Base.@_inline_meta` at $f:$l is deprecated, use the no-arg `@inline` annotation instead.", :var"@_inline_meta")
```

How about this change ?

Reply:
I would do
```julia
@deprecate var"@_inline_meta" var"@inline" false
```